index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+            href="https://fonts.googleapis.com/css2?family=Inter&family=Roboto:ital,wght@0,100;0,300;0,700;1,100&display=swap"
+            rel="stylesheet"
+    />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
+    <title>Document</title>
+</head>
+<body>
+<nav class="navbar">
+    <a href="">Kinyera Amos</a>
+    <div class="mobile_menu">
+        <div class="menu-btn">
+            <div class="humbergure"></div>
+        </div>
+    </div>
+</nav>
+<div class="background-image">
+    <h1 class="title">KINYERA AMOS</h1>
+    <h3 class="heading">Software Developer</h3>
+    <p>
+        I can help you build a product ,
+        feature or website Look through
+        some of my work and experience! If
+        you like what you see and have a
+        project you need coded,  don’t
+        hestiate to contact me.
+    </p>
```

Comment:
I think you should change the font size from 18px to 21px. It would make your code look more organised and user-friendly to the view.

Reply:
- [ ] Kindly don't ask for changes in an imperative way like `you should` and please add the reason why the reviewee had to do the changes. for this example you can say:
`please add the correct value of this paragraph font to make it match the given design`